eng/targets/Settings.props

```diff
@@ -17,6 +17,7 @@
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
     <ToolsetPackagesDir>$(RepoRoot)build\ToolsetPackages\</ToolsetPackagesDir>
     <RoslynPortableTargetFrameworks>netcoreapp3.1;net472</RoslynPortableTargetFrameworks>
+    <RoslynLibraryTargetFrameworks>netcoreapp3.1;netstandard2.0</RoslynLibraryTargetFrameworks>
```

Comment:
I prefer that we not use MSBuild properties for our `TargetFramework` groups. It makes the projects themselves harder to read. I do understand I am the one who created the original groups but ended up deleting them later because I didn't really think it was worth the cost here.